cax/cax.json

```diff
@@ -69,10 +71,15 @@
     "receive": "scp",
     "hostname": "tegner-login-1.pdc.kth.se",
     "directory": "/cfs/klemming/projects/xenon/xenon1t/data",
+    "dir_raw": "/cfs/klemming/projects/xenon/xenon1t/data",
+    "dir_root": "/cfs/klemming/projects/xenon/xenon1t/processed",
     "upload_options": [],
     "username": "bobau",
     "download_options": [
       "midway-login1"
+    ],
+    "pax_versions": [
+      "v4.10.0"
```

Comment:
I haven't "enabled" the processing in Stockholm. I use this json definition to define the folder where later the root files are stored. See: data_mover.py, line 125